client/src/app/app.component.spec.ts

```diff
@@ -1,15 +1,26 @@
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { TestBed } from '@angular/core/testing';
+import { AngularFireAuthModule } from '@angular/fire/compat/auth';
 import { RouterTestingModule } from '@angular/router/testing';
 import { AppComponent } from './app.component';
+import { AuthService } from './services/auth.service';
 
 
+let authStub = {
```

Comment:
```suggestion
const authStub = {
```
Vu que tu ne le modifie pas.